src/site/components/banner_alerts.drupal.liquid

```diff
@@ -0,0 +1,76 @@
+{% for entity in alert.entities %}
+{% assign dismissStatus = entity.fieldAlertInheritanceSubpages %}
+{% assign alertType = entity.fieldAlertType %}
+
+{% if alertType = "information" %}
+{% assign alertType = "info" %}
+{% endif %}
+
+{% assign region = path | split: "/" | first | prepend: "/" %}
+{% assign lastArg = path | split: "/" | last | prepend: "/"%}
+
+{% assign outputStatus = false %}
+{% assign emailUpdates = "https://www.va.gov/" %}
+{% assign statusUrl = "https://www.va.gov/" %}
+
+{% for vamc in entity.fieldBannerAlertVamcs %}
+
+{% if region == vamc.entity.fieldOffice.entity.entityUrl.path %}
+
+{% assign outputStatus = true %}
+
+{% if dismissStatus == true and lastArg != region and lastArg != "/operating-status" %}
+{% assign outputStatus = false %}
+{% endif %}
+{% assign emailUpdates = vamc.entity.fieldOffice.entity.fieldLinkFacilityEmergList.uri %}
+{% assign statusUrl = vamc.entity.entityUrl.path %}
+{% endif %}
+{% endfor %}
+
+{% if outputStatus == true %}
+<div aria-labelledby="usa-alert-heading-{{ entity.entityId }}"
+  class="usa-alert-full-width dismissable-option-header usa-alert-full-width-{{ entity.fieldAlertType }}"
```

Comment:
Looks like we're using `alertType` elsewhere. Should this follow suit?

Reply:
Updated @cvalarida :+1: